packages/frontend-2/codegen.ts

```diff
@@ -26,15 +26,15 @@ const config: CodegenConfig = {
         enumsAsConst: true,
         useTypeImports: true,
         fragmentMasking: false,
-        dedupeFragments: true,
+        dedupeFragments: false,
```

Comment:
huh? i dont think this is OK, I remember adding this specifically because the server could not handle duplicate fragments being sent over